Assignment3/Assignment3.py

```diff
@@ -0,0 +1,82 @@
+# Importing necessary libraries
+import sys
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+from pyspark.ml import Transformer
+
+# Constructing a transformer
+
+class Rollingaverage(Transformer):
+
```

Comment:
Implement the transformer. You can refer to the professor's code for making a custom transformer.
Link: https://teaching.mrsharky.com/sdsu_fall_2020_lecture05.html#/7/7/3

Reply:
Yeah done that!